src/libraries/sendtohelix.proj

```diff
@@ -89,7 +89,7 @@
   <Target Name="CompressRuntimeDirectory"
           Inputs="@(_RuntimeInputs);@(TestArchiveRuntimeDependency)"
           Outputs="$(TestArchiveRuntimeFile)"
-          Condition="'$(BuildAllConfigurations)' != 'true' and '$(TargetsMobile)' != 'true'">
+          Condition="'$(BuildAllConfigurations)' != 'true' and '$(TargetOS)' != 'iOS' and '$(TargetOS)' != 'tvOS' and '$(TargetOS)' != 'Android'">
```

Comment:
As a suggestion we could introduce a property here in sendtohelix.proj to know if we're targeting any of these and share it for all the conditions.

Reply:
I always thought it's `$(TargetsMobile)`?

Reply:
TargetsMobile includes wasm as part of it so this si trying to do `TargetsMobile!=true or TargetOS == Browser` basically

Reply:
yeah I think we can go with `TargetsMobile != true or TargetOS == Browser` like we do elsewhere